### PR TITLE
data:RishabhDhawad

### PR DESCRIPTION
**Pull request by RishabhDhawad**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3633"></a>

### CODE REVIEW

**Review by RishabhDhawad — COMMENTED** [review]
I have updated my initial commit and it's now according to the JSON format

File: data/RishabhDhawad.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/rishabhdhawad/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/rishabhdhawad/",
+      "icon": "FaInstagram"
+    },
```
Comment (ChinmayMhatre) [inline data/RishabhDhawad.json:34]:
```suggestion
    }
```
Please remove this comma

File: data/RishabhDhawad.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  ],
```
Comment (ChinmayMhatre) [inline data/RishabhDhawad.json:35]:
```suggestion
  ]
```
Please remove this comma

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
I have suggested some changes as to why your PR might be failing

**Comment by RishabhDhawad** [comment]
Thank you so much for pointing it out, Chinmay. Also, I have updated the JSON file.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**